src-java/floodlight-service/floodlight-api/src/main/java/org/openkilda/messaging/payload/switches/InstallIslDefaultRulesCommand.java

```diff
@@ -47,18 +47,27 @@
     @JsonProperty("dst_port")
     private int dstPort;
 
+    @JsonProperty("multitable_mode")
+    private boolean multitableMode;
```

Comment:
I don't see were we use this field?

Reply:
Fixed - as it was a bug in the code. The field is needed and used in RecordHandler.doInstallIslDefaultRule method.